recipes/jsonrpcserver/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "jsonrpcserver" %}
+{% set version = "4.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 56f3ab21aad908920538ab32e0a608c1d39ccb0e00c7a112f13484e53b434542
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >3
+    - pip
+  run:
+    - python >3
+
+test:
+  imports:
+    - jsonrpcserver
+
+about:
+  home: https://github.com/bcb/jsonrpcserver
+  license: MIT
+  license_family: MIT
+  license_file: https://github.com/bcb/jsonrpcserver/blob/master/LICENSE
```

Comment:
It seems that the license is not being packaged, because of that, you need to package it manually. Please see
https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

Reply:
Made a new release which packages the license, and updated the recipe.